numpy/typing/mypy_plugin.py

```diff
@@ -16,6 +16,7 @@
   Without the plugin the type will default to `ctypes.c_int64`.
 
   .. versionadded:: 1.22
+  .. deprecated:: 2.3
```

Comment:
Technically inside the bullet, but the `.. versionadded::` for the plugin isn't inside the plugin docstring, so maybe it doesn't matter.